internal/mongodoc/doc.go

```diff
@@ -200,6 +200,12 @@ type BaseEntity struct {
 	// set of resource names holding the currently published resource
 	// version for that channel and resource name.
 	ChannelResources map[params.Channel][]ResourceRevision
+	
+	// NoIngest is set to true when a charm or bundle has been uploaded
+	// with a POST request. Since the ingester only uses PUT requests
+	// at present, this signifies that someone has taken over control from
+	// the ingester.
+	NoIngest bool	`bson:",omitempty"`
```

Comment:
I don't think I like the name NoIngest. It seems to represent the wrong thing. I think I'd prefer a name that represents what happened and then the can-ingest endpoint can decide based on that whether it's allowed. So perhaps this should be "Posted" or "Pushed".